File: jdate.js
function jdate() {
    const now=new Date();
    const month=now.getMonth()+1;
    const day=now.getDate();
    const year=now.getFullYear();
    return `${month}.${day}.${year}`;
}

// One liner:
// `${new Date().getMonth()+1}.${new Date().getDate()}.${new Date().getFullYear()}`

File: jdate.py
import datetime

def jdate():
    now=datetime.datetime.now()
    year=now.strftime("%y")
    month=str(int(now.strftime("%m"))) #str(int($_)) so no
    day=str(int(now.strftime("%d")))
    return f'{month}.{day}.{year}'

print(jdate())
